mathics/builtin/patterns/rules.py

```diff
@@ -0,0 +1,563 @@
+# -*- coding: utf-8 -*-
+"""
+Define, apply and compiling rules.
+
+<url>
+:WMA link:
+https://reference.wolfram.com/language/guide/Rules.html</url>
+
+
+Rules are a basic element in the evaluation process. Every Definition in WL \
```

Comment:
Everywhere else, we use the term Mathics3 (or the older deprecated Mathics) instead of WL.

Reply:
Note that the macro is still \Mathics  not \Mathics3. 

In general, you are not looking at the generated documentation.